src/mtp_cmd_handler.c

```diff
@@ -843,6 +846,27 @@ static void __reset_device(mtp_handler_t *hdlr)
 	_cmd_hdlr_send_response_code(hdlr, PTP_RESPONSE_OK);
 }
 
+
+static void __send_partial_object(mtp_handler_t *hdlr)
+{
+	mtp_uint32 h_obj = 0;
+	off_t offset = 0;
+	mtp_uint32 data_sz = 0;
+	mtp_uint32 send_bytes = 0;
+	data_blk_t blk = { 0 };
+	mtp_uchar *ptr = NULL;
```

Comment:
The compiler complains in this section about unused variables.

Reply:
Sure will handle this and update this commit.